tests/openvino/test_exporters_cli.py

```diff
@@ -961,49 +962,93 @@ def test_exporters_cli_full_quantization(
                 expected_fake_nodes_per_model,
             )
 
-    def test_exporters_cli_int4_with_local_model_and_default_config(self):
+    @parameterized.expand(
+        [
+            (
+                "falcon-40b",
+                "tiiuae/falcon-7b-instruct",
```

Comment:
no need for the model id, let's continue using MODEL_NAMES[model_type]

Reply:
Do you have a suggestion how to avoid using `model_id`? In the test we patch `_name_or_path` at `config.json` with the `model_id`. This is needed for the default config to match up and avoids adding a test element to `_DEFAULT_4BIT_WQ_CONFIGS`. 

Unfortunately, for the newly added test, I had to add a separate config for the test model anyways because there is no data-free full quantization default config and collecting 300 samples would take too much time. Also, there is no way to patch the config-retrieval logic because the export is run in a separate process.

Reply:
my bad, thought the model_id is the one we're gonna load (was surprised it's a 7b model). I see now that a "model_id" is necessary to trick the quantizer to apply a certain recipe !